test/unit/native_deps/native_deps_test.bzl

```diff
@@ -18,12 +18,26 @@ def _assert_argv_contains(env, action, flag):
         "Expected {args} to contain {flag}".format(args = action.argv, flag = flag),
     )
 
+def _assert_argv_contains_prefix_suffix(env, action, prefix, suffix):
```

Comment:
Could you also update all other tests? AFAIU all of create types should get -Lnative flags, right?

Reply:
Indeed! Done :)